app/assets/javascripts/oxalis/store.js

```diff
@@ -391,31 +432,28 @@ export const defaultState: OxalisState = {
     logoUrl: null,
   },
   tracing: {
-    annotationId: "",
-    boundingBox: null,
-    createdTimestamp: 0,
-    userBoundingBox: null,
-    type: "readonly",
-    name: "",
-    version: 0,
-    isPublic: false,
-    tracingId: "",
-    tracingType: "View",
-    restrictions: {
-      branchPointsAllowed: false,
-      allowUpdate: false,
-      allowFinish: false,
-      allowAccess: true,
-      allowDownload: false,
-      somaClickingAllowed: false,
-      allowedModes: ["orthogonal", "oblique", "flight"],
+    ...initialAnnotationInfo,
+    readOnly: {
+      ...initialAnnotationInfo,
```

Comment:
The initialAnnotationInfo is part of the tracing property and part of the readOnly property of that tracing. Why is that?